src/App.js

```diff
@@ -1,10 +1,26 @@
-import React from 'react'
-import Login from './components/Login.js';
+import React from 'react';
+import Login from './components/Login/Login.js';
+import { useState } from 'react';
+import Dashboard from './components/Dashboard/Dashboard.js';
 
 function App() {
+
+  const [isLoggedIn, setIsLoggedIn] = useState(false)
+  const [activeUser, setActiveUser] = useState('')
+
+  function handleLogin(username) {
+    setIsLoggedIn(!isLoggedIn);
+    setActiveUser(username)
+  }
+
   return (
     <div className="App">
-      <Login />
+      {
+      !isLoggedIn
+      ? <Login onLogin={handleLogin}/>
+      // : <p>Herzlich willkommen, {activeUser}</p>
+      : <Dashboard letActiveUser={activeUser}/>
```

Comment:
`letActiveUser` sounds like a declaration of a variable. Why not keep the name `activeUser`?
```suggestion
      : <Dashboard activeUser={activeUser}/>
```